src/main/java/it/gov/pagopa/debtposition/repository/PaymentPositionRepository.java

```diff
@@ -40,10 +43,12 @@ public interface PaymentPositionRepository extends JpaRepository<PaymentPosition
     //Optional<PaymentPosition> findByPaymentOptionOrganizationFiscalCodeAndPaymentOptionNav(String organizationFiscalCode, String nav); // search only by nav
     
     // TODO #naviuv: temporary regression management: search by nav or iuv
+    @Lock(LockModeType.PESSIMISTIC_READ)
```

Comment:
add commento with following ref https://www.baeldung.com/jpa-pessimistic-locking#lock-modes
could be useful

Reply:
done